packages/orchestration/test/exos/local-orchestration-account-kit.test.ts

```diff
@@ -131,9 +134,19 @@ test('delegate, undelegate', async t => {
   // 2. there are no return values
   // 3. there are no side-effects such as assets being locked
   await E(account).delegate(validatorAddress, bld.units(999));
-  // TODO get the timer to fire so that this promise resolves
-  void E(account).undelegate(validatorAddress, bld.units(999));
-  t.pass();
+  const undelegateP = E(account).undelegate(validatorAddress, bld.units(999));
+  timer.advanceTo(
+    TimeMath.coerceTimestampRecord(
+      Number(UNBOND_PERIOD_SECONDS + maxClockSkew),
+      timer.getTimerBrand(),
+    ),
+    'advance timer to end of unbonding period',
+  );
+  t.is(
+    await undelegateP,
+    undefined,
+    'undelegate returns void after completion_time',
```

Comment:
you can test that it actually waited that long by racing with `timer.wakeAt(expectedFullfillment - 1n)`

Reply:
Updated to include, ty. Also realized I don't need to provide a brand to the zone manual timer.